sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -4043,6 +4043,16 @@ object functions {
   def array_compact(column: Column): Column = withExpr {
     ArrayCompact(column.expr)
   }
+    /**
```

Comment:
nit: The indentation of comments is different from others

Reply:
@LuciferYang thanks. I've made the updates. There were some failures that seem to be unrelated to this change. I reran the workflow and seems to have succeeded this time.

Reply:
All Passed now :)